server/utils/emailService.js

```diff
@@ -0,0 +1,33 @@
+const nodemailer = require('nodemailer');
```

Comment:
emailService is already implemented, no need of implementing a new one.

Reply:
The email service that was already in your code, was not for signup. It was for, forgot password. I have added a new page, for entering the OTP, once the user presses the signup button.